### PR TITLE
First draft of Summon command

### PR DESCRIPTION
**Pull request by Christopher-Chianelli**

### CODE REVIEW

**Comment by salmanfs815** [comment]
2 suggestions:
* summoner should already be in channel
* summoned should not already be in channel

**Comment by ianthpun** [comment]
im loving this sudden burst of work coming in 👍 keep it up

File: vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java [hunk 1]
```diff
@@ -78,7 +116,7 @@ public static void main( String[] args )
...
...r( new com.oldterns.vilebot.handlers.admin.Ops() ).addListener( new Quit() ).addListener( new AnswerQuestion() ).addListener( new Ascii() ).addListener( new ChatLogger() ).addListener( new Church() ).addListener( new Countdown() ).addListener( new Decide() ).addListener( new Excuses() ).addListener( new FakeNews() ).addListener( new Fortune() ).addListener( new GetInfoOn() ).addListener( new Help() ).addListener( new ImageToAscii() ).addListener( new Inspiration() ).addListener( new Jaziz() ).addListener( new Jokes() ).addListener( new Kaomoji() ).addListener( new Karma() ).addListener( new KarmaRoll() ).addListener( new LastMessageSed() ).addListener( new LastSeen() ).addListener( new Markov() ).addListener( new News() ).addListener( new Omgword() ).addListener( new Ops() ).addListener( new QuotesAndFacts() ).addListener( new RemindMe() ).addListener( new RockPaperScissors() ).addListener( new Trivia() ).addListener( new Ttc() ).addListener( new TwitterCorrection() ).addListener( new UrlTitleAnnouncer() ).addListener( new UrlTweetAnnouncer() ).addListener( new Userlists() ).addListener( new UserPing() ).addListener( new Weather() ).addListener( new Summon() ).buildConfiguration();
```
Comment (cuijulian) [inline vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java]: Put `.addListener( new Summon() )` in alphabetical order with the other listeners.

File: vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java [hunk 2]
```diff
@@ -78,7 +116,7 @@ public static void main( String[] args )
...
```
Comment (cuijulian) [inline vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java]: `.addListener( new Summon() )` should be in alphabetical order with the other listeners.

**Pull request merged**